src/main/java/org/umssdiplo/automationv01/core/managepage/smartHouse/SHEmployee.java

```diff
@@ -0,0 +1,96 @@
+package org.umssdiplo.automationv01.core.managepage.smartHouse;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+import org.umssdiplo.automationv01.core.utils.PropertyAccessor;
+
+public class SHEmployee extends BasePage {
```

Comment:
you need a constructor in this case. Because when you are going to load this page, you need to wait for an WebElement especific.
CommonEvents.isVisible(yourWebElment).